website/src/components/OrganismTableSelector.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import type { FC } from 'react';
+
+import { routes } from '../routes/routes.ts';
+import type { Metadata, InputField } from '../types/config';
+import { groupFieldsByHeader } from '../utils/groupFieldsByHeader';
+import IwwaArrowDown from '~icons/iwwa/arrow-down';
+
+export type OrganismMetadata = {
+    key: string;
+    displayName: string;
+    metadata: Metadata[];
+    inputFields: InputField[];
+};
+
+type Props = {
+    organisms: OrganismMetadata[];
+};
+
+const OrganismTableSelector: FC<Props> = ({ organisms }) => {
```

Comment:
The name seems a bit too generic, would something like `OrganismMetadataTableSelector` be better?